recipe/meta.yaml

```diff
@@ -1,52 +1,58 @@
 {% set name = "libsemigroups_pybind11" %}
-{% set version = "0.1.1" %}
+{% set version = "1.0.1" %}
 
 package:
-  name: libsemigroups_pybind11
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/libsemigroups/libsemigroups_pybind11/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: a51878de09443d9e632a0c502aaa47b314a54ee793731c20f1874896890dfecb
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/libsemigroups_pybind11-{{ version }}.tar.gz
+  sha256: 3fc2dca669fe4468d524d293099df5dd84074b6049f88ce222ea560b36f5bdec
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
   host:
-    - python
+    - setuptools >=64
+    - setuptools-scm >=8
+    - libsemigroups >=3.1.3
+    - packaging
     - pip
-    - libsemigroups
-    - pkgconfig >=0.29.2
-    - packaging >=20.4
-    - pybind11 >=2.6
+    - pkgconfig
+    - pybind11
+    - python
   run:
+    - numpy
     - python
-    - libsemigroups >=2.0.2
-    - pybind11 >=2.6
+    - python-graphviz
+    - typing_extensions
 
 test:
   imports:
     - libsemigroups_pybind11
+  commands:
+    - pip check
+    - pytest
   requires:
-    - pytest=6.2.4
+    - python 3.9
+    - pip
+    - pytest
   source_files:
     - tests/
-  commands:
-    - pytest -vv tests/test_*.py
 
 about:
-  home: https://libsemigroups.github.io/
-  license: GPL-3.0-or-later
-  license_family: GPL
-  license_file: LICENSE
-  summary: 'Python package for using the C++ library libsemigroups via pybind11'
+  home: https://libsemigroups.github.io/libsemigroups_pybind11/
   dev_url: https://github.com/libsemigroups/libsemigroups_pybind11
+  summary: A python package for the libsemigroups C++ library.
+  license: GPL-3.0-only
+  license_file: LICENSE
 
 extra:
   recipe-maintainers:
-    - chinmayanagpal
```

Comment:
Do we know that @chinmayanagpal does not want to maintain this package anymore?

Reply:
I'm quite certain that @chinmayanagpal doesn't want to maintain the package anymore; he is a former colleague of mine that moved away from the development of this project a few years ago.

That said, if you think it's more appropriate to keep Chin as a maintainer until we get explicit confirmation, I'm happy to do that too.

Reply:
Also if possible @wolfv can you please give me the ability to merge prs into this repo, I have this ability for the libsemigroups-feedstock but not for the python bindings in this repo

Reply:
Thanks for confirming this @chinmayanagpal (Also, hello! Hope all is well with you too!). In this case, are we okay to merge @wolfv?